src/Models/Laravel/MarketplaceSale.php

```diff
@@ -28,6 +28,7 @@ class MarketplaceSale extends BaseModel
      */
     protected $fillable = [
         'marketplace_listing_id',
+        'listing_id',
```

Comment:
It is to solve the same issue we had with `collection_id` on beams. After syncing `marketplace_listing_id` will no longer be the same, so instead of using it, we'll be using `listing_id` as foreign key

Reply:
But isn't `marketplace_listing_id` the blockchain id? That's the internal id?

Reply:
Yes it's internal id, listing_id is from blockchain

Reply:
Hmmm ok, that's confusing. Maybe should we use the same standard as on core we we have for example

`collection_id` as foreign key (internal)
`collection_chain_id` as the blockchain id

I think that's it, right?

Reply:
Yes that is right, unfortunately it was named as `listing_id` on `marketplace_listings` table so to be consistent i use `listing_id` for this table as well